### ISSUE TITLE
How can definiert ListColumn as StaticColumn ?

### ISSUE THREAD

**Issue by riBN** [outworkers/phantom#650]
I write the schema of my table and I want that the column manager will be static.  

```
 object managers  extends SetColumn[Int](this)  with StaticColumn[Set[Int]]
```

but it is not work because when I write Desc table users,  I have :  
 managers set<int>

**Comment by alexflav23** [comment]
This should work. What version of phantom are you using?

```
 object managers  extends SetColumn[Int](this)  with StaticColumn
```

**Comment by riBN** [comment]
@alexflav23  it is't wok I must the table define in StaticColumn 
phantom version is: 2.11-1.29.6

**Comment by alexflav23** [comment]
@riBN We no longer support 1.29.6, the current version is 2.6.2. I can try and patch that for you, let me see what the problem is and how it can be fixed.

**Comment by riBN** [comment]
@alexflav23 for the name I can define a staticColumn. it is work but only the "SetType" can not be define.

**Comment by alexflav23** [comment]
Hi @riBN I'm not sure what you are saying, if you just want to file the bug in German please do so 🇩🇪

**Comment by riBN** [comment]
@alexflav23 I need to define "manager" as columnStatic so I need help ;)

**Comment by alexflav23** [comment]
Hi @riBN,

As you are using a very old version of phantom, you will need to upgrade to `2.6.4` for us to get the fixed static collection columns to you. Are you planning to update soon? We are deploying the fix to 2.6.5, but there will be work required on your part to upgrade to the latest phantom before you can benefit from the fix.

Regards,
Flavian

**Comment by riBN** [comment]
@alexflav23  ok,  thanks

**Comment by alexflav23** [comment]
Hi @riBN This is fixed in 2.7.5, but you will need to upgrade to benefit from the fix.
